OpenPKW/ResultResponseDto.m

```diff
@@ -0,0 +1,48 @@
+//
+//  ResultResponseDto.m
+//  OpenPKW
+//
+//  Created by Piotr Tobolski on 10.05.2015.
+//  Copyright (c) 2015 OpenPKW. All rights reserved.
+//
+
+#import "ResultResponseDto.h"
+
+@implementation ResultResponseDto
+
+- (instancetype)initWithDictionary:(NSDictionary *)dictionary {
+    if (self = [self init]) {
+        _permittedPeopleCount = [dictionary[@"uprawnionych"] unsignedIntegerValue];
+        _votingPeopleCount = [dictionary[@"glosujacych"] unsignedIntegerValue];
+        _cardsValidCount = [dictionary[@"kartWaznych"] unsignedIntegerValue];
+        _votesInvalidCount = [dictionary[@"glosowNieWaznych"] unsignedIntegerValue];
+        _votesValidCount = [dictionary[@"glosowWaznych"] unsignedIntegerValue];
+        
+        NSUInteger candidateNumber = 1;
+        NSMutableDictionary *votesForCandidate = [[NSMutableDictionary alloc] init];
+        while (1) {
```

Comment:
Bez różnicy, ponieważ YES === (signed char)1 ale jeśli uważasz, że ładniej wygląda to możesz zmienić, mi się nie chce :-P